packages/now-go/index.js

```diff
@@ -55,7 +55,7 @@ async function build({ files, entrypoint }) {
 
   // check if package name other than main
   const packageName = parseFunctionName.split(',')[1];
-  const isGoModExist = await pathExists(`${entrypointDirname}/go.mod`);
+  const isGoModExist = await pathExists(`${entrypointDirname}${sep}go.mod`);
```

Comment:
```suggestion
  const isGoModExist = await pathExists(join(entrypointDirname, 'go.mod'));
```

Let's use the more traditional join when concatenating paths 😄